articulate/orson
# Bugfix cc safari14

This PR contains a fix for closed captioning not working/displaying in Safari 14.  

Apparently something changed in Safari 14 - we're not sure exactly what, but we dug up [this stack overflow article](https://stackoverflow.com/questions/64162432/ios-14-and-macos-safari-14-broke-captions-using-video-js) and the fix worked.

**Testing Steps**
See the [rise-frontend PR](https://github.com/articulate/rise-frontend/pull/2642) for testing steps.

**Links**
Issue: articulate/rise-authoring#181
rise-frontend: https://github.com/articulate/rise-frontend/pull/2642
